client/src/pages/Home/Home.tsx

```diff
@@ -4,18 +4,62 @@ import StoreList from "src/components/StoreList";
 import TextInputField from "src/components/TextInputField";
 import UserHeader from "src/components/UserHeader";
 import DebugBar from "./DebugBar";
-import { User, emptyUser, Store, GMapPlace } from "src/interfaces";
-import { getUserInfo } from "src/pages/Actions";
-import { Container } from "@material-ui/core";
-import { isDebug } from "src/config";
+import SearchBar from "src/components/SearchBar";
+import { User, emptyUser, Store, GMapPlace, GeoLocation } from "src/interfaces";
+import {
+  getUserInfo,
+  getGeoLocation,
+  getStoresByLocation,
+  getNearbyPlacesTest,
+} from "src/pages/Actions";
+import { Container, Typography } from "@material-ui/core";
+import { isDebug, google } from "src/config";
 import { transformGMapPlaceToStore } from "src/transforms";
 
 function Home(props: any) {
   const [userid, setUserid] = useState("");
   const [curUser, setCurUser] = useState<User>(emptyUser());
   const [stores, setStores] = useState<Store[]>([]);
   const [testPlaces, setTestPlaces] = useState<GMapPlace[]>([]);
-  const [placeStore, setPlaceStore] = useState<Store[]>([]);
+  const [placeStores, setPlaceStores] = useState<Store[]>([]);
+  const [useLocation, setUseLocation] = useState(false);
+  const [curGeoLocation, setCurGeoLocation] = useState<GeoLocation | null>(
+    null
+  );
+
+  const SECRET_TRIGGER = "hungry";
```

Comment:
what is this for? if this is just used for your testing, you should not check it in

Reply:
easter egg? Typing `hungry` will send a request to get nearby restaurants via Places API